reinforcement_learning/rl_unity_ray/rl_unity_ray.ipynb

```diff
@@ -483,9 +483,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from sagemaker.tensorflow.serving import Model\n",
+    "from sagemaker.tensorflow.model import TensorFlowModel\n",
     "\n",
-    "model = Model(model_data=estimator.model_data,\n",
+    "model = TensorFlowModel(model_data=estimator.model_data,\n",
```

Comment:
Is this the SDK 2.0 specific?

Reply:
Yes, v2 changes the Tensorflow model class.